x-pack/packages/kbn-elastic-assistant/impl/assistant/api/conversations/bulk_update_actions_conversations.ts

```diff
@@ -139,7 +139,13 @@ export const bulkUpdateConversations = async (
     toasts?.addError(error.body && error.body.message ? new Error(error.body.message) : error, {
       title: i18n.translate('xpack.elasticAssistant.conversations.bulkActionsConversationsError', {
         defaultMessage: 'Error updating conversations {error}',
-        values: { error },
+        values: {
+          error: error.message
```

Comment:
Found an issue when handling error, please find the item 4 in https://github.com/elastic/kibana/pull/187705#discussion_r1674452482